ReactiveCocoa/Swift/DynamicProperty.swift

```diff
@@ -5,71 +5,70 @@ import enum Result.NoError
 /// types, including generic types when boxed via `AnyObject`).
 private protocol ObjectiveCRepresentable {
 	associatedtype Value
-	static func extract(from representation: Any) -> Value
-	static func represent(_ value: Value) -> Any
+	static func extract(from representation: Any?) -> Value
```

Comment:
If you pass an optional to `Any`, it would be recognised as an opaque box (`SwiftValue`).